src/buf.yaml.template

```diff
@@ -7,23 +7,14 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0.
 
+# ${generation-comment}
+
 version: v1
 breaking:
   use:
     - WIRE
   ignore:
-    # does currently not require backward-compatibility
-    - cluster-client
```

Comment:
I think we still want to mark files in the currently listed crates with `// buf breaking: ignore`. @petrosagg and me will then double-check that this does not include `Proto$T` types that are written by `persist`.